src/App.tsx

```diff
@@ -1,9 +1,38 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
 
+type Movies = {
+  title: string,
+  description: string,
+  imgUrl: string,
+  imdbUrl: string,
+  imdbId: string,
+};
+
 export const App: React.FC = () => {
+  const [query, setQuery] = useState('');
+
+  function findMovies(movies: Movies[], input: string) {
+    return movies.filter(
+      ({ title, description }) => {
+        if (input === '') {
+          return movies;
+        }
+
+        const regex = new RegExp(input.trim(), 'ig');
```

Comment:
If there is no input you shouldn't to do filter loop, but return movies right away. And it's better to trim input to make sure the string is really empty
```suggestion
  function findMovies(movies: Movies[], input: string) {
    if (!input.trim()) {
        return movies;
    }
        
    return movies.filter(({ title, description }) => {
        const regex = new RegExp(input.trim(), 'ig');
```

Reply:
Thanks, fixed it!